app/javascript/packs/document-capture.tsx

```diff
@@ -78,31 +67,12 @@ const trackEvent: typeof baseTrackEvent = (event, payload) => {
   });
 };
 
-(async () => {
-  const backgroundUploadURLs = getBackgroundUploadURLs();
-  const isAsyncForm = Object.keys(backgroundUploadURLs).length > 0;
-
+(() => {
   const formData: Record<string, any> = {
     document_capture_session_uuid: appRoot.getAttribute('data-document-capture-session-uuid'),
     locale: document.documentElement.lang,
   };
 
-  let backgroundUploadEncryptKey;
-  if (isAsyncForm) {
-    backgroundUploadEncryptKey = await window.crypto.subtle.generateKey(
-      {
-        name: 'AES-GCM',
-        length: 256,
-      },
-      true,
-      ['encrypt', 'decrypt'],
-    );
-
-    const exportedKey = await window.crypto.subtle.exportKey('raw', backgroundUploadEncryptKey);
```

Comment:
did we have any polyfills for the subtle crypto library? if so should we remove them?

Reply:
> did we have any polyfills for the subtle crypto library? if so should we remove them?

I don't think we have any polyfills anymore between removing support for Internet Explorer and upgrading to Node.js v16+. We _might_ be able to remove the assignment of the `window.crypto` global in the [test setup](https://github.com/18F/identity-idp/blob/47a39265a4c4ba2678d2035eba1d4aa5582b6f8d/spec/javascript/spec_helper.js#L30), but I'd also be fine to keep it so that the test global generally looks & acts like a real DOM global.